src/Tools/Microsoft.VisualStudio.Templates.AssemblyInfo.Wizard/AssemblyInfoTemplateWizard.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using EnvDTE;
+
+using Microsoft.VisualStudio.TemplateWizard;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+public partial class AssemblyInfoTemplateWizard : IWizard
+{
+    private static readonly string CSharp_Minimal_AssemblyInfoTemplate = $@"using System.Runtime.InteropServices;
+
+// In SDK-style projects the assembly information is automatically
+// generated on build.
+
+// The information generated comes from the Package page on the Project Properties.
+
+// Other attributes can be specified here as necessary
```

Comment:
```suggestion
// Other attributes can be specified here as necessary.
```

Other lines end in a full stop. Ditto below for the `Guid` line, then again in `VB_Minimal_AssemblyInfoTemplate`.

Reply:
Can I claim that the lack of full stop on the `Guid` line is a back compat issue? :D